tests/tt_eager/python_api_testing/sweep_tests/test_configs/ci_sweep_tests/broken_grayskull/pytorch_eltwise_power_fp_test.yaml

```diff
@@ -9,7 +9,7 @@ test-list:
         num-samples: 64
         args-sampling-strategy: "all"
       env:
-        TT_METAL_SLOW_DISPATCH_MODE: ["", "1"]
+        TT_METAL_SLOW_DISPATCH_MODE: [""]
```

Comment:
What's the reason these sweep tests are being modified regarding slow vs fast dispatch testing?

Reply:
@tt-aho In WHB0 test is aborted if fast dispatch is enabled, hence the fast dispatch setting is disabled

Reply:
This file is for gs (broken_grayskull dir)

Reply:
Similarly, the one in broken_wormhole is expected to fail right? So should that have been modified to skip FD test?

Reply:
@tt-aho Yes GS test is modified to test only FD and WHB0 test modified to test SD alone